Docs/source/usage/parameters.rst

```diff
@@ -414,11 +415,20 @@ Besides, for profiles that depend on spatial coordinates (the plasma momentum di
 
 The parser reads python-style expressions between double quotes, for instance
 ``"a0*x**2 * (1-y*1.e2) * (x>0)"`` is a valid expression where ``a0`` is a
-user-defined constant (see below) and ``x`` and ``y`` are spatial coordinates. The names are case sensitive. The factor
+user-defined constant (see above) and ``x`` and ``y`` are spatial coordinates. The names are case sensitive. The factor
 ``(x>0)`` is ``1`` where ``x>0`` and ``0`` where ``x<=0``. It allows the user to
 define functions by intervals.
 Alternatively the expression above can be written as ``if(x>0, a0*x**2 * (1-y*1.e2), 0)``.
 
+Integer Expressions
+^^^^^^^^^^^^^^^^^^^
+
+Expressions with integers allow the all of the usual mathematical and logical operations.
```

Comment:
```suggestion
Expressions with integers allow all of the usual mathematical and logical operations.
```

Thanks for adding this! (just a minor typo above)

And can we also do something like
```
my_constants.L = 1e-5
my_constants.dx = 1e-8
my_constants.theta = 0.4
amr.n_cell = L/dx L/dx*sin(theta) L/dx/tan(theta)
```

and it rounds down to the nearest integers or is that forbidden and we're limited to using integers throughout the operations?

Reply:
That is the important question to ask. As it is now, it can only handle integers since it uses the IParser. Your example won't work (since the integer parser won't know how to read in the my_constants). I'll rethink this and go back to what I had originally thought about (before seeing the IParser), treating everything as real numbers until the final value, and convert that to an integer (rounding to the nearest integer).

Reply:
Ok thanks, that would be more general indeed (although the current implementation with the IParser is already useful by itself).

Reply:
I've pushed the updated version. In the end, this is a smaller, simpler change anyway.